roles/base/tasks/redhat_tasks.yml

```diff
@@ -10,6 +10,7 @@
     update_cache: true
     state: latest
   with_items:
+    - gcc
```

Comment:
just curious, are these dependencies for nfs? I have seen @jainvipin get worried when we install things like compilers in `base` role so just confirming this is really needed in `base` :)